### ISSUE TITLE
As a group coordinator, I want to post a message to my group’s stream on behalf of the group.

### ISSUE THREAD

**Issue by lmb244** [lmco/eurekastreams#130]
1. Add Post as Announcement section in the following design:
   http://www.eurekastreams.org/docs/product-specification/features/view-manage-group-profile/
2. Add updates to the Personal Settings, Notifications, and Emails to add the post as group announcement feature are described in the following design:
   http://www.eurekastreams.org/docs/product-specification/features/manage-receive-notifications/

**Comment by cstephe** [comment]
Should group announcements notify group members?

**Comment by lmb244** [comment]
Yes.  There will be a .9.2 release that will include group members receiving a notification any time something is posted to the group he/she is a member of.  So that functionality will actually be out there before this.  Thanks

**Comment by lmb244** [comment]
It would be useful to have a 'broadcast' message feature where users get a pop-up message (sent to everyone, just a group, etc.) for key alerts. a simple example would be for Eureka itself... let's say the system is going down for unplanned maintenence... all users get a pop-up explaining what is going on. This feature would be valuable to group coordinators as well.
